hoomd/hpmc/pytest/test_quick_compress.py

```diff
@@ -13,20 +13,35 @@
 # here that require preprocessing
 valid_constructor_args = [
     dict(trigger=hoomd.trigger.Periodic(10),
-         target_box=hoomd.Box.from_box([10, 10, 10])),
+         target_box=hoomd.variant.box.Constant(hoomd.Box.from_box([10, 10,
+                                                                   10]))),
```

Comment:
Let's keep at least one case where the constructor arg is still a regular box since it should be able to accept both.